src/dataflow/src/source/mod.rs

```diff
@@ -139,30 +139,15 @@ where
     pub key: K,
     /// The record's value
     pub value: V,
-    /// The position in the source, if such a concept exists (e.g., Kafka offset, file line number)
-    pub position: Option<i64>,
+    /// The position in the source (e.g., Kafka offset, file line number, monotonic increasing
```

Comment:
Since you're here, this is the position in the partition described by `partition` in the source